.github/ISSUE_TEMPLATE/optimize-epic.md

```diff
@@ -1,10 +1,10 @@
 ---
 
-name: Optimize Epic
-about: This issue is used to track the progress of an Optimize Epic
-title: ''
-labels: type:epic
-assignees: ''
+name: Optimize PDP Epic
+about: Summary of tasks/features to implement PDP epic.
```

Comment:
I am not sure that this is an improvement, it doesn't better describe its purpose

Reply:
the reason is as mentioned in the answer to the other comment